apps/src/music/views/SoundsPanel.tsx

```diff
@@ -327,7 +327,11 @@ const SoundsPanel: React.FunctionComponent<SoundsPanelProps> = ({
       >
         <div id="hidden-item" tabIndex={0} role="button" />
         {showSoundFilters && (
-          <div id="sounds-panel-top" className={styles.soundsPanelTop}>
+          <div
+            id="sounds-panel-top"
+            className={styles.soundsPanelTop}
+            data-theme="Dark"
```

Comment:
Can you use the theme from ThemeWrapper that gets set in the lab entry points instead of hard coding?

Reply:
Also just calling out that the Lab2 theme/themewrapper logic was added before we had DSCO theming. Now that we have the DSCO theme system and the equivalent context wrapper, we should probably just replace it since it's basically the same as our Lab2 theme wrapper. 

Regardless though, yeah it would probably make sense to grab the theme from context rather than hardcoding (even though we don't really have any actual light mode support for music lab).

Reply:
@sanchitmalhotra126 do you have any examples handy of using the DSCO / point me to the context wrapper for that? I am doing some changes in the generic dialog right now that require some overrides for the button styles since we don't have dark mode for those components yet and I didn't realize we had DSCO theming already

Reply:
O nvm I found ThemeContext.

Reply:
Yep that's the one (not to be confused Lab2's ThemeContext 😅), though worth calling that only a few DSCO components support theming right now

Reply:
After a good discussion, it looks like the right plan is to replace Lab2's `ThemeContext` with the DSCO equivalent in a future piece of work.  Thanks both!